react/PushClientBanner/styles.styl

```diff
@@ -0,0 +1,35 @@
+@require '../../stylus/components/button.styl'
+@require '../../stylus/settings/breakpoints.styl'
+
+.c-banner-client
+    position relative
+
+    +small-screen('min')
+        margin-top     -1.875rem
+        margin-bottom  1.875rem
+
+    .c-btn-clientMobile
+        @extend $button-client-mobile
+        margin  0
```

Comment:
Nit: could the margin be part of `$button-client-mobile`?

Reply:
I guess yeah.